govuk-one-login/authentication-api
# AUT-2704: unused registration email otp feature flag cleanup

## What

cleanup PR as **removeRetryLimitForRegistrationEmailCodeEntry** feature flag is no longer required.

## How to review

1. Code Review
2. Deploy to sandpit with `./deploy-sandpit.sh -a`
3. Ensure that block screen appears after 5 attempts as before PR



## Checklist
